test/e2e/tempopb/tempopb.go

```diff
@@ -0,0 +1,31 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package tempopb
+
+// this is a package copied from tempo project,
+// EG only use pb struct for testing purpose.
```

Comment:
to avoid any ambiguity wrt referencing , i'd suggest unmarshalling to `map[string]interface{}` so we rm this

Reply:
`map[string]interface{}` is untyped and not firendly to use, another option is switch to use `Zipkin`.

BTW, I don't think pb struct should be under MPL license.